extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1241,11 +1241,14 @@ describes.realWin('amp-story', {
     });
   });
   describe('amp-story branching', () => {
+
+    beforeEach(() => {toggleExperiment(win, 'amp-story-branching', true);});
+    afterEach(() => {toggleExperiment(win, 'amp-story-branching', false);});
```

Comment:
Here you could use either `before/afterEach()` or `before/after()`. The first one is going to be executed before/after each test, when the first one is going to be executed only once before all the tests, and then once after all the tests. I think we might want the latter in this very use case :)

Reply:
Synced offline, but the tests always fail when using `before/after`, reproduce-ably. Seems to be a bug!